### ISSUE TITLE
Convert the activation service to js-ng

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldfoundation/tft-stellar#199]
TODO:
- [x] Overwrite the old code with the new one 
- [x] Deploy on testnet

**Comment by robvanmieghem** [comment]
blocked on https://github.com/threefoldtech/js-sdk/issues/1120

**Comment by AhmedSa-mir** [comment]
js-ng activation-service package code here: https://github.com/AhmedSa-mir/activation-service/tree/master/activation_service

**Comment by robvanmieghem** [comment]
@AhmedSa-mir Thanks but there still seems to be a few things missing like
- gevent pool to serialize the account activations
- the wallet to take to activate accounts
- the location rewrite of the url 

The pacga.toml also declares isauth and isadmin as true, which should not be the case

**Comment by xmonader** [comment]
@robvanmieghem Hi AhmedSam-ir is on sick leave today, I prepared the package and will see if @ranatrk can't take over 

the code is open on https://github.com/threefoldfoundation/tft-stellar/pull/216

**Comment by robvanmieghem** [comment]
thanks @xmonader

**Comment by robvanmieghem** [comment]
the interface of the service is not backwards compatible, the postadata in jsx used to be {"args":{...}} while the args wrapper is no longer needed. I like it better but it will break current clients.

**Comment by robvanmieghem** [comment]
there is also another issue: #220

**Comment by ranatrk** [comment]
Cases handled in last fix:
- Valid cases
    request body:
    ```
    {"address":"GBKXUZJLFUPIVX7SGEJHEUDCM7QVPYRQ7LVQFPUFXYMWBQC4IOWEEY42"}
    ```
    response 200:
    ```
    "{\"args\": {\"activation_code\": \"abcd\", \"address\": \"GBKXUZJLFUPIVX7SGEJHEUDCM7QVPYRQ7LVQFPUFXYMWBQC4IOWEEY42\", \"phonenumbers\": [\"+1234567890\"]}}"
    ```
- Invalid cases
    - case 1
        request body:
        ```
        {"address":"GBKXUZJLFUPIVX7SGEJHEUDCM7QVPYRQ7LVQFPUFXYMWBQC4IOWEEY42"}
        ```
        response 400:
        ```
        {
        "error": "This address is not new"
        }
        ```
    - case 2
        request body:
        ```
        {"address":"aaa"}
        ```
        response 400:
        ```
        {
        "error": "Address is invalid"
        }
        ```
    - case 3
        request body:
        ```
        {"address":"<newaddress>"}
        ```
        response 400:
        ```
        {
        "error": "Address is invalid"
        }
        ```
    - case 4 (removed couple of characters from the beginning)
        request body:
        ```
        {"address":"JLFUPIVX7SGEJHEUDCM7QVPYRQ7LVQFPUFXYMWBQC4IOWEEY42"}
        ```
        response 400:
        ```
        {
        "error": "Address is invalid"
        }
        ```
    - case 5 (removed couple of characters from the end)
        request body:
        ```
        {"address":"GBKXUZJLFUPIVX7SGEJHEUDCM7QVPYRQ7LVQFPUFXYMWBQC4"}
        ```
        response 400:
        ```
        {
        "error": "Address is invalid"
        }
        ```

**Comment by robvanmieghem** [comment]
It's already a lot better but I thinbk you misunderstood my previous comment. It's not the response that should have it wrapped in `"args":` but the request so it works identical to the jsx service

**Comment by ranatrk** [comment]
@robvanmieghem oh sorry, will adjust it now to work with
```
 {"args":{"address": <wallet_address>}
```
and also accept a string` <wallet_address>`


and will change the return like before without the args

**Comment by robvanmieghem** [comment]
Has been modified already to use sponsored reserves and deployed to production
